python/ray/autoscaler/_private/monitor.py

```diff
@@ -148,18 +144,7 @@ def __init__(
         stop_event: Optional[Event] = None,
         retry_on_failure: bool = True,
     ):
-        if not use_gcs_for_bootstrap():
-            # Initialize the Redis clients.
-            redis_address = address
-            self.redis = ray._private.services.create_redis_client(
-                redis_address, password=redis_password
```

Comment:
Should we add some comments somewhere that `redis_password` is a deprecated arg?

Reply:
@DmitriGekhtman  this will be removed in the following cleanup PR. My plan is 1) remove bootstrap related code; 2) remove cpp bootstrap related code; 3) remove all redis related code in python except starting redis server;

I think at 3) this will be deleted.

Is that OK?

Reply:
I'd prefer it if we leave the arg in and state in comments that it's deprecated.